snapcraft/internal/project_loader/_config.py

```diff
@@ -21,7 +21,7 @@
 import re
 
 import jsonschema
-from typing import Set  # noqa: F401
+from typing import List, Set  # noqa: F401
```

Comment:
can you remove the noqa from here and make sure these are used explicitly?

Reply:
done